### PR TITLE
Model/add.unit test

### PR DESCRIPTION
**Pull request by JosueRodriguezGarcia**
Hi guys, 
Pull request for audio modificationPlease redirect yourself to keep Jesus working.

### CODE REVIEW

File: src/main/java/com/fundation/webservice/model/AudioResponse.java [hunk 1]
```diff
@@ -21,63 +21,113 @@
...
+    private String aBitRate;
     private String aChannel;
-    private String aRate;
-
     private String checksum;
 
+    /**
+     * Method constructor  of the audio response class initial the all variable.
+     *
+     * @param fileName
```
Comment (alszla) [inline src/main/java/com/fundation/webservice/model/AudioResponse.java:32]: Lines 32-40: Please add descriptions for each paramerters

File: src/main/java/com/fundation/webservice/model/AudioResponse.java [hunk 2]
```diff
@@ -21,63 +21,113 @@
...
-    public String getaRate() {
-        return aRate;
-    }
-
+    /**
+     * Method that return the checksum in format MD5 or SHA-1.
+     *
+     * @return checksum of the file.
+     */
     public String getChecksum() { return checksum; }
```
Comment (LimbertVargas) [inline src/main/java/com/fundation/webservice/model/AudioResponse.java]: if "return checksum" is only in one code line then you should convert all the "return" lines in one line code contrary case change this line like the other that contain "return".

Comment (JosueRodriguezGarcia) [reply src/main/java/com/fundation/webservice/model/AudioResponse.java]: fix

Comment (LimbertVargas) [reply src/main/java/com/fundation/webservice/model/AudioResponse.java]: ok... that's good.

**Pull request merged**